lib/matplotlib/__init__.py

```diff
@@ -1251,21 +1251,24 @@ def interactive(b):
     """
     Set interactive mode to boolean b.
 
-    If b is True, then draw after every plotting command, e.g., after xlabel
+    If b is True, then draw after every plotting command, e.g., after xlabel.
     """
     rcParams['interactive'] = b
 
 
 def is_interactive():
-    'Return true if plot mode is interactive'
```

Comment:
"... whether to redraw after each command", as for `interactive()` (the current docstring is pretty uninformative)